fish_prompt.fish

```diff
@@ -393,16 +393,15 @@ end
 
 function __bobthefish_prompt_rubies -d 'Display current Ruby (rvm/rbenv)'
   [ "$theme_display_ruby" = 'no' ]; and return
-  set -l ruby_version
-  if type rvm-prompt >/dev/null 2>&1
+  if which rvm-prompt >/dev/null 2>&1
     set ruby_version (rvm-prompt i v g)
-  else if type rbenv >/dev/null 2>&1
-    set ruby_version (rbenv version-name)
+  else if which rbenv >/dev/null 2>&1
+    set ruby_version (cat .ruby-version 2>/dev/null)
```

Comment:
Yeah, I'm all for improving the actual library where possible, but if we can save all our users 30ms every time they show their prompt in the meantime, I wouldn't complain about that either :)